etc/picongpu/juwels-jsc/booster_picongpu.profile.example

```diff
@@ -0,0 +1,126 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+#   - using NF - number of fields, use the last group available
+#export proj=$(groups | awk '{print $(NF)}')
+export proj=ea_picongpu
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="vim"
+
+# Set up environment, including $SCRATCH and $PROJECT
+jutil env activate -p $proj
+
+# General modules #############################################################
+#
+# Requires module stages to be loaded
+OTHERSTAGES=/p/software/juwelsbooster/otherstages
+module purge
+
+module use $OTHERSTAGES
+module load Stages/2020
+
+module load GCC/9.3.0
+module load CMake/3.18.0
+module load ParaStationMPI/5.4.7-1
+module load mpi-settings/default
+#module load mpi-settings/CUDA 
+module load Python/3.8.5
+
+module load Boost/1.74.0
```

Comment:
@Anton-Le I think PIConGPU does not support boost 1.74.0 - please see: https://picongpu.readthedocs.io/en/latest/install/dependencies.html#boost
Or @psychocoderHPC is this documentation outdated?

Reply:
Yes, documentation is outdated. The upper supported version was not increased.

1.65.1 is minimum https://github.com/alpaka-group/alpaka/blob/eb31bcd89bae022fd2d818008af8f68d30bcf482/cmake/alpakaCommon.cmake#L156